Portfolio mobile.html

```diff
@@ -7,18 +7,196 @@
+                <li>html</li>
+                <li><img src="./images/Rectangle%2070.png" alt="split"/></li>
+                <li>bootstrap</li>
+                <li><img src="./images/Rectangle%2070.png" alt="split"/></li>
+                <li>Ruby</li>
+            </ul>
+        <button class="button" type="button">See Project</button>
+    </div>
+    <img class="Placeholder" src="./images/Img%20Placeholder.png" alt="Image place holder"/>
+    </article>
+    </section>
+
+    <section class="Project_work3">
+        <article class="article">
+        <img class="Placeholder" src="./images/Img%20Placeholder.png" alt="Image place holder"/>
+        <div class="workform">
+        <h5 class="posted">Multi-Post Stories</h5>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+            <ul class="techstack">
+                <li>css</li>
+                <li><img src="./images/Rectangle%2070.png" alt="split"/></li>
+                <li>html</li>
+                <li><img src="./images/Rectangle%2070.png" alt="split"/></li>
+                <li>bootstrap</li>
+                <li><img src="./images/Rectangle%2070.png" alt="split"/></li>
+                <li>Ruby</li>
+            </ul>
+        <button class="button" type="button">See Project</button>
+    </div>
+    </article>
+    </section>
+
+    <section class="Project_work4">
+        <article class="article">
+        <div class="workform">
+        <h6 class="posted">Multi-Post Stories</h6>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+            <ul class="techstack">
+                <li>css</li>
+                <li><img src="./images/Rectangle%2070.png" alt="split"/></li>
+                <li>html</li>
+                <li><img src="./images/Rectangle%2070.png" alt="split"/></li>
+                <li>bootstrap</li>
+                <li><img src="./images/Rectangle%2070.png" alt="split"/></li>
+                <li>Ruby</li>
+            </ul>
+        <button class="button" type="button">See Project</button>
+    </div>
+    <img class="Placeholder" src="./images/Img%20Placeholder.png" alt="Image place holder"/>
+    </article>
+    </section>
+    <footer class="Contact-form">
+        <h2>Let's chat!</h2>
+        <p>If you have an application you are interested in developing, 
+                a feature that you need built or a project that needs coding. 
+                I'd love to help with it 
+        </p>
+        <form class="form-info" action="https://formspree.io/f/mzbooyno" method="post">
```

Comment:
- [ ] You can consider reducing the box shadow and adding more padding to the text area so your design is similar to the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220601_211154](https://user-images.githubusercontent.com/25227022/171496331-b77e8516-9c65-4e41-9f0b-e7b5747b4a3b.png)|![Screenshot_20220601_211213](https://user-images.githubusercontent.com/25227022/171496345-8983dbd3-bf0e-4579-a08b-a55a682139c7.png)|